demo/src/main/java/com/ringpublishing/gdpr/demo/SplashActivity.java

```diff
@@ -56,28 +57,31 @@ private void showAppContent()
         // Here you can check if user agreed for all available vendors
         if(RingPublishingGDPR.getInstance().areVendorConsentsGiven())
         {
-            initializeExternalLibraries();
+            // If you app uses SDK's from vendors, which are not part of the official TCF 2.0 vendor list
+            // you can use this flag to check if you can enable / initialize them as user agreed for all vendors
+            initializeSplashExternalLibraries();
         }
         startActivity(new Intent(this, MainActivity.class));
         finish();
-        // If you app uses SDK's from vendors, which are not part of the official TCF 2.0 vendor list
-        // you can use this flag to check if you can enable / initialize them as user agreed for all vendors
+
     }
 
-    private void initializeExternalLibraries()
+    private void initializeSplashExternalLibraries()
     {
-        // Here you can initialize external libraries, because consent screen was already visited by user.
+        // Here you can initialize external libraries initialized in your Splash screen,
+        // because consent screen was already visited by user.
```

Comment:
consent screen has been already presented to user.

Reply:
Changed